Azure.ResourceManager.Core/ResourceGroupContainer.cs

```diff
@@ -89,6 +89,7 @@ public async override Task<ArmOperation<ResourceGroup>> StartCreateAsync(string
         /// List the resource groups for this subscription.
         /// </summary>
         /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="P:System.Threading.CancellationToken.None" />. </param>
+        /// <returns> A response with the <see cref="Pageable{ResourceGroup}"/> operation for this resource. </returns>
```

Comment:
Use the standard response text for Pageable
https://github.com/m-nash/azure-proto-sdk/blob/master/XmlDocStandard.md#pageable